github-actions/open_pr.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
Would be nice to have an overview on what this does.

Reply:
Fixed with 72a90d5.